src/java.base/share/classes/sun/security/ec/ECPrivateKeyImpl.java

```diff
@@ -246,4 +237,4 @@ private void readObject(ObjectInputStream stream)
         throw new InvalidObjectException(
                 "ECPrivateKeyImpl keys are not directly deserializable");
     }
-}
+}
```

Comment:
Please restore this newline from upstream (but that's not present upstream in jdk22).

Reply:
Done.